examples/bitmap_operation/example_bitmap_operation.cpp

```diff
@@ -24,17 +24,14 @@ int main()
         // Second way to do
         method2();
     }
-    catch( const imageException & ex ) {
-        // uh-oh, something went wrong!
+    catch( imageException & ex ) { // uh-oh, something went wrong!
```

Comment:
Please follow the structure which I wrote in the original issue:
```cpp
catch( const std::exception & ex ) { // uh-oh, something went wrong!
    std::cout << "Exception " << ex.what() << " raised. Closing the application..." << std::endl;
    return 1;
}
catch( ... ) { // uh-oh, something terrible happen!
    std::cout << "Generic exception raised. Closing the application..." << std::endl;
    return 2;
}

std::cout << "Application ended correctly." << std::endl;
return 0;
```

Just copy-paste the code in all files :)
Only for below files we don't need `std::cout << "Application ended correctly." << std::endl;`:
```bash
examples/cuda/pinned_memory/pinned_memory.cpp
examples/qt_framework/qimage_display/main.cpp
examples/raspicam/main.cpp
examples/thread_pool/example_thread_pool.cpp
```
Above files output or show something so we don't need to output last message which is an indicator that application completed it's work.

Reply:
Sorry for that, I was in hurry with last commit and totally messed stuff.
I also removed unreachable "return 0;" from qt_framework/qimage_display/main.cpp

Reply:
No blame on you :) We all make mistakes ;)

Reply:
True, but I should have followed the rule: first think, than do, than commit ;}

Reply:
I fixed that,